src/fastoad/model_base/tests/test_atmosphere.py

```diff
@@ -155,12 +155,12 @@ def test_speed_conversions():
         [270, 266.0652, 150.4692],
         [400, 394.1707, 222.9173],
     ]
-    expected_CAS = [  # currently unused
-        [100.0, 98.5799, 56.3313],
-        [200.0, 197.3624, 116.1973],
-        [270, 161.8971, 266.6984],
-        [400, 395.5782, 252.6355],
-    ]
+    # expected_CAS = [  # currently unused
+    #     [100.0, 98.5799, 56.3313],
+    #     [200.0, 197.3624, 116.1973],
+    #     [270, 161.8971, 266.6984],
+    #     [400, 395.5782, 252.6355],
+    # ]
```

Comment:
Since it is unused and since the Atmosphere class will soon be deprecated in the core, can't we just delete it ?

Reply:
I don't mind. As you say, we should remove the class and the associated tests when we'll decide to rely only on stdatm, so I don't consider that as important.